README.md

```diff
@@ -5,41 +5,64 @@ because running audius should look like this
 ./audius
 ```
 
-## run
+## install
+
+**x86**
+```
+curl -o audius https://raw.githubusercontent.com/AudiusProject/dot-slash-audius/main/bin/audius-x86 && chmod +x audius
+```
+
+**arm**
```

Comment:
You can get creative and run a small bash script that checks for the architecture, not necessary though. Would just eliminate the need for different install commands.
```
#!/bin/bash
if [[  $(uname -m) == "arm64" ]]; then
    ./audius-arm
else
    # assume x86 for now
    ./audius-x86
fi

```